rosbridge_library/test/capabilities/test_subscribe.py

```diff
@@ -37,8 +37,12 @@ def dummy_cb(self, msg):
         pass
 
     def test_update_params(self):
-        """Adds a bunch of random clients to the subscription and sees whether
-        the correct parameters are chosen as the min"""
+        """
+        Test the update_params method of the Subscription class.
+
+        Adds a bunch of random clients to the subscription and sees whether
```

Comment:
Should this be "Add" per the other changes?

Reply:
I only cared about the summary line to be written in imperative mood as per PEP257 guidelines. In the longer description which explains inner workings of a function or side effects, I left it in descriptive mood.